packages/query/package.json

```diff
@@ -42,9 +42,5 @@
     "@hcengineering/core": "^0.6.32",
     "@hcengineering/analytics": "^0.6.0",
     "fast-equals": "^5.0.1"
-  },
```

Comment:
we need to keep this publishable

Reply:
removed because now it is not published, will add proper config